app/assets/javascripts/initializers/initializeBaseUserData.js

```diff
@@ -108,6 +108,21 @@ function setCurrentUserToNavBar(user) {
       .getElementsByClassName('js-header-menu-admin-link')[0]
       .classList.remove('hidden');
   }
+
+  // If the user has policies in their data, let's iterate through them and start
+  if (user.policies) {
+    user.policies.forEach(function (policy) {
+      if (
+        policy.resource_type == 'Article' &&
+        policy.action == 'create' &&
```

Comment:
```suggestion
        policy.resource_type === 'Article' &&
        policy.action === 'create' &&
```
`==` makes me twitchy in JS 😄

Reply:
The `==` is the Rubyist in @jeremyf 😆

Reply:
>I thought the property equality operator of JS was `¯\_(ツ)_/¯`.

This is going to be my official stance from now on 😆